lib/elixir/test/elixir/fixtures/calendar/holocene.exs

```diff
@@ -12,33 +14,63 @@ defmodule Calendar.Holocene do
                    microsecond: microsecond, calendar: __MODULE__}
   end
 
+  @impl true
   def date_to_string(year, month, day) do
     "#{year}-#{month}-#{day} (HE)"
   end
 
+  @impl true
   def naive_datetime_to_string(year, month, day, hour, minute, second, microsecond) do
-    "#{year}-#{month}-#{day} #{Calendar.ISO.time_to_string(hour, minute, second, microsecond)} (HE)"
+    "#{year}-#{month}-#{day}" <>
+      Calendar.ISO.time_to_string(hour, minute, second, microsecond) <> " (HE)"
+  end
+
+  @impl true
+  def datetime_to_string(year, month, day,
+                         hour, minute, second, microsecond,
+                         _time_zone, zone_abbr, _utc_offset, _std_offset) do
+    "#{year}-#{month}-#{day}" <>
+      Calendar.ISO.time_to_string(hour, minute, second, microsecond) <> " #{zone_abbr} (HE)"
   end
 
+  @impl true
   defdelegate time_to_string(hour, minute, second, microsecond), to: Calendar.ISO
 
+  @impl true
   def day_rollover_relative_to_midnight_utc(), do: {0, 1}
 
+  @impl true
   def naive_datetime_from_iso_days(entry) do
     {year, month, day, hour, minute, second, microsecond} =
       Calendar.ISO.naive_datetime_from_iso_days(entry)
     {year + 10000, month, day, hour, minute, second, microsecond}
   end
 
+  @impl true
   def naive_datetime_to_iso_days(year, month, day, hour, minute, second, microsecond) do
     Calendar.ISO.naive_datetime_to_iso_days(year - 10000, month, day, hour, minute, second, microsecond)
   end
 
+  @impl true
   defdelegate time_from_day_fraction(day_fraction), to: Calendar.ISO
 
+  @impl true
   defdelegate time_to_day_fraction(hour, minute, second, microsecond), to: Calendar.ISO
 
+  @impl true
   defdelegate leap_year?(year), to: Calendar.ISO
 
+  @impl true
   defdelegate days_in_month(year, month), to: Calendar.ISO
+
+  @impl true
+  defdelegate day_of_week(year, month, day), to: Calendar.ISO
+
+  @impl true
+  def valid_date?(year, month, day) do
```

Comment:
On a related note: ISO also requires year >= 0, which means some Holocene dates are not representable. So writing property based tests we'd have a failure:

```
iex> date = ~D[2017-01-01]
iex> date |> Date.convert!(Calendar.Holocene) |> Date.convert!(Calendar.ISO)
~D[2017-01-01]

iex> {:ok, date} = Date.new(2017, 1, 1, Calendar.Holocene)
iex> date |> Date.convert!(Calendar.ISO) |> Date.convert!(Calendar.Holocene)
** (FunctionClauseError) no function clause matching in :calendar.last_day_of_the_month/2

    The following arguments were given to :calendar.last_day_of_the_month/2:

        # 1
        -7983

        # 2
        1
```

So: looking forward to having negative dates (#6294) and property-based tests in Elixir itself!